sdv/constraints/tabular.py

```diff
@@ -956,3 +956,40 @@ def reverse_transform(self, table_data):
         table_data[self._columns] = transformed_data
 
         return table_data
+
+
+class Unique(Constraint):
+    """Ensure that each value for a specified column/group of columns is unique.
+
+    This constraint is provided a list of columns, and guarantees that every
+    unique combination of those columns appears at most once in the sampled
+    data.
+
+    Args:
+        columns (str or list[str]):
+            Name of the column(s) to keep unique.
+    """
+
+    def __init__(self, columns):
+        self.columns = [columns] if isinstance(columns, str) else columns
```

Comment:
I would do it the other way around: If `columns` is not a `list`, make it a list of a single element.
Otherwise different column name types, such as `int`s will not work.